peer/brontide.go

```diff
@@ -358,6 +365,57 @@ type Config struct {
 	Quit chan struct{}
 }
 
+// The pingManager is a structure that is designed to manage the internal state
+// of the ping pong lifecycle with the remote peer. We assume there is only one
+// ping outstanding at once.
+type pingManager struct {
```

Comment:
Same comment here re `godoc` comment prefix. Will leave this as the last comment so the rest of the diff isn't spammed.